users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -281,6 +283,7 @@ def run_specaug_scf():
                 returnn_args=dict(
                     batch_size=3500,
                     specaug_mask_sorting=True,
+                    specaug_after_first_layer=True,
```

Comment:
As discussed in other threads, please move this to a separate PR.
```suggestion
```

Reply:
Done